python/orca/src/bigdl/orca/learn/pytorch/pytorch_pyspark_estimator.py

```diff
@@ -175,7 +176,8 @@ def __init__(
             cluster_info=self._get_cluster_info(sc),
             **local_init_params)
 
-        self.state_dict = self.driver_runner.get_state_dict()
+        if self.model_creator or self.model_class:
+            self.state_dict = self.driver_runner.get_state_dict()
```

Comment:
Is the logic here correct? This is the init method of PySparkEstimator and model_class will always be None?

Reply:
yes correct, some state_dict like "model" will be None in this case, so we needn't get state_dict here. Instead, `self.state_dict` will get value after `estimator.fit` or `load`.

Reply:
I mean self.model_class will always be None as it is in the init method?

Reply:
sure.